ui/lib/apps/SlowQuery/utils/useColumn.tsx

```diff
@@ -127,7 +134,9 @@ export function useDBColumn(
 export function useSuccessColumn(
   _rows?: { success?: number }[] // used for type check only
 ): IColumn {
-  const { t } = useTranslation()
+  // !! Wrong usage, hooks only can be used inside the Function Component
+  // !! Can't be used in normal function
```

Comment:
This wording is not correct exactly, hooks can be used in the custom hook as well. we need to avoid `useXxxColumn()` to become the custom hook.